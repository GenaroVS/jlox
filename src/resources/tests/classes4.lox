class Obj {
    init() {
        return "Something";
    }
}

print "init method returning value";
var o = Obj();
