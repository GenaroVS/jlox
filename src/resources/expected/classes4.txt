[line 3] ERROR at 'return': Can't return a value from a class initializer.
